spec/helpers/admin/help_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the Admin::HelpHelper. For example:
+#
+# describe Admin::HelpHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe Admin::HelpHelper, type: :helper do
+end
```

Comment:
We can probably delete these auto-generated files if we don't expect to fill them in. Same with the SCSS files above.

Reply:
I'll delete these.